core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1860,8 +1863,17 @@ private[spark] class DAGScheduler(
             s"(attempt ${failedStage.latestInfo.attemptNumber}) running")
         } else {
           failedStage.failedAttemptIds.add(task.stageAttemptId)
+          val ignoreStageFailure = ignoreDecommissionFetchFailure &&
+            isExecutorDecommissioned(bmAddress)
+          if (ignoreStageFailure) {
+            logInfo("Ignoring fetch failure from $task of $failedStage attempt" +
```

Comment:
This log message might be misleading or wrong when `disallowStageRetryForTest` is true.